### ISSUE TITLE
Rows not scrolling in beta-26

### ISSUE THREAD

**Issue by tgolbs** [schrodinger/fixed-data-table-2#482]
Hi there,
when we started developing our application we started using beta-8. We now wanted to upgrade to the newest beta but do have the strange behaviour that the table will not scroll its content. It shows the shadows in the correct way though.

## Expected Behavior
We would expect that the rows will be rerendered on scroll.

## Current Behavior
No rerender happens on scroll.

## Steps to Reproduce (for bugs)
https://codesandbox.io/embed/distracted-vaughan-crqe6?fontsize=14

**Comment by pradeepnschrodinger** [comment]
We had a couple people report this on the latest versions of beta, but none had a reproducible example we could start looking into, and for some reason it worked just fine in our app which uses latest FDT.

Strangely this also passed the tests, and nothing seemed wrong in the examples. So not sure why this happens. Anyway I'll definitely start looking into this, now that there's a reproducible example.

Thanks for reporting with a sandbox link 😄

**Comment by tgolbs** [comment]
You're welcome. I am quite happy, that is not some misconfiguration on my side, because there were times i strongly questioned myself :D
We tried switching to a newer beta version before (beta-23) and the same behaviour was observed but we did not have time to look into it. So the change responsable for it must not be in the latest version update.

**Comment by pradeepnschrodinger** [comment]
Hey, thanks for the information. Yea, I feel it was introduced through #460, which went into `v1.0.0-beta.21` (see this [comment](https://github.com/schrodinger/fixed-data-table-2/pull/460#issuecomment-506913766) and continued discussion on the PR)

I do notice one thing. The latest beta works just fine with React `15.6.2`, the one in FDT's [package.json](https://github.com/schrodinger/fixed-data-table-2/blob/v1.0-beta/package.json#L48), which is also the one used in the [site](http://schrodinger.github.io/fixed-data-table-2/), and in our product. So that's probably why we could never reproduce the bug.

In the codepen, if you rollback `react` and `react-dom` version from `16.8.6` to `16.0.0-alpha.2` or less, it works well. I'm trying to figure out why it makes a difference though 😕 

**As for what happens**:
It seems the render method for the rows are never being called after the initial table render.
Basically, all rows under `FixedDataTableBufferedRows` (which makes up the content) are never rendered more than once, no matter how much you scroll. `FixedDataTableBufferedRows` itself, however, is called properly during every scroll, so basically it's not rendering it's children, although it's clearly returned... Very weird. 

We'll look more into this. @wcjordan, @apar03 , @mycrobe, any of you know why this might have occurred? Is it some weird optimization introduced in later versions? Or are any of the lifecycle methods dead due to React version upgrade?

**Comment by pradeepnschrodinger** [comment]
@tgolbs , we released a fix for this. Please try latest beta [v1.0.0-beta.28](https://github.com/schrodinger/fixed-data-table-2/releases/tag/v1.0.0-beta.28). 

Again, thanks so much for the reproducible sandbox. Really helped in finding out the [cause](https://github.com/schrodinger/fixed-data-table-2/pull/483#issue-317929604).

I'm closing this issue as it seems fixed, but feel free to continue the discussion if needed.

**Comment by tgolbs** [comment]
@pradeepnschrodinger Thank you very much! We upgraded to beta-28 and it now works like a charm.
